ci/values.yaml

```diff
@@ -16,14 +16,6 @@ clickhouseOperator:
   namespace: default
   storage: 2Gi
```

Comment:
now if this one is changed we won't catch it - can we remove this one too, e.g. set the default to be 3Gi (2Gi is needed for Clickhouse without any data)